### PR TITLE
renaming

### PR DESCRIPTION
**Pull request by se3000**
- rename confirmation config names
- move github docs from `.github/` to `docs/`
- rename cltest fixtures to factorires
- move gui building go file into gui directory

### CODE REVIEW

File: examples/ropsten/RopstenNode.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Use the following environment variables as an example to configure your node for
     ROOT=~/.ropsten
     ETH_URL="ws://localhost:18546"
     ETH_CHAIN_ID=3
-    TX_MIN_CONFIRMATIONS=2
+    MIN_OUTGOING_CONFIRMATIONS=2
     TASK_MIN_CONFIRMATIONS=2
```
Comment (thodges-gh) [inline examples/ropsten/RopstenNode.md]: I think this one would also need to be called `MIN_INCOMING_CONFIRMATIONS`

File: adapters/adapter.go [hunk 2]
```diff
@@ -83,7 +83,7 @@ func For(task models.TaskSpec, store *store.Store) (AdapterWithMinConfs, error)
 	}
 	wa := MinConfsWrappedAdapter{
 		Adapter:                 ac,
-		ConfiguredConfirmations: store.Config.TaskMinConfirmations,
+		ConfiguredConfirmations: store.Config.MinOutgoingConfirmations,
```
Comment (dimroc) [inline adapters/adapter.go]: Perhaps my understanding is incorrect, but aren't many of these swapped?

We are waiting on enough confirmations to have the confidence to run tasks with side effects:
`TaskMinConfirmations` -> `MinIncomingConfirmations`

We are waiting on enough confirmations to believe that our sent (outgoing) eth transaction as been written to the blockchain:
`TxMinConfirmations` -> `MinOutgoingConfirmations`

File: internal/cltest/cltest.go [hunk 3]
```diff
@@ -71,17 +71,17 @@ func NewConfigWithWSServer(wsserver *httptest.Server) *TestConfig {
...
-			EthGasBumpThreshold:  3,
-			EthGasPriceDefault:   *big.NewInt(20000000000),
-			DatabasePollInterval: store.Duration{Duration: time.Millisecond * 500},
+			LogLevel:                  store.LogLevel{Level: zapcore.DebugLevel},
+			RootDir:                   rootdir,
+			BasicAuthUsername:         Username,
+			BasicAuthPassword:         Password,
+			ChainID:                   3,
+			MinOutgoingConfirmations:  6,
+			MinIncomingConfirmations: 0,
```
Comment (rupurt) [inline internal/cltest/cltest.go]: +1 extra spacing before `0`

File: store/config.go [hunk 4]
```diff
@@ -21,24 +21,24 @@ import (
...
+	RootDir                  string          `env:"ROOT" envDefault:"~/.chainlink"`
+	Port                     string          `env:"CHAINLINK_PORT" envDefault:"6688"`
+	GuiPort                  string          `env:"GUI_PORT" envDefault:"6689"`
+	BasicAuthUsername        string          `env:"USERNAME" envDefault:"chainlink"`
+	BasicAuthPassword        string          `env:"PASSWORD" envDefault:"twochains"`
+	EthereumURL              string          `env:"ETH_URL" envDefault:"ws://localhost:8546"`
+	ChainID                  uint64          `env:"ETH_CHAIN_ID" envDefault:"0"`
+	ClientNodeURL            string          `env:"CLIENT_NODE_URL" envDefault:"http://localhost:6688"`
+	MinIncomingConfirmations uint64          `env:"MIN_OUTGOING_CONFIRMATIONS" envDefault:"12"`
+	MinOutgoingConfirmations uint64          `env:"MIN_INCOMING_CONFIRMATIONS" envDefault:"0"`
```
Comment (rupurt) [inline store/config.go]: struct members should match env vars i.e.

- `MinIncomingConfirmations` = `MIN_INCOMING_CONFIRMATIONS`
- `MinOutgoingConfirmations` = `MIN_OUTGOING_CONFIRMATIONS`

**Pull request merged**